app/style/app.css

```diff
@@ -78,7 +88,7 @@ gaia-header #upload-link {
 .install-list .install-button {
```

Comment:
We have both ```.install-list .item``` and ```.install-list > .item```. We should remove the latter as it's being overridden anyways.